cmd/devstream/develop.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/merico-dev/stream/internal/pkg/develop"
+	"github.com/merico-dev/stream/pkg/util/log"
+)
+
+var name string
+
+var developCMD = &cobra.Command{
+	Use:   "develop",
+	Short: "Develop is used for develop a new plugin",
+	Long: `Develop is used for develop a new plugin.
+eg.
+- dtm develop create-plugin --name=YOUR-PLUGIN-NAME`,
+	Run: developCMDFunc,
+}
+
+func developCMDFunc(cmd *cobra.Command, args []string) {
+	if err := validateArgs(args); err != nil {
+		log.Fatal(err)
+	}
+
+	developAction := develop.Action(args[0])
+	log.Debugf("The develop action is: %s", developAction)
+	if err := develop.BranchAction(developAction); err != nil {
```

Comment:
I think we can have a better name here for "Branch Action". I understand here the purpose is to switch on the action, but the name "branch action" sounds weird. Maybe "ExecuteAction(developAction)" is good enough?

Reply:
agree